spec/defines/foreman_dynflow_worker_spec.rb

```diff
@@ -42,21 +42,58 @@ class { 'foreman':
             {queues: ['default', 'remote_execution']}
           end
 
+          let(:expected_content) do
+            <<~CONTENT
+              ---
+              :concurrency: 1
+              :queues:
+              - default
+              - remote_execution
+            CONTENT
+          end
+
           it { should compile.with_all_deps }
           it {
             should contain_file('/etc/foreman/dynflow/test_worker.yml')
               .with_ensure('file')
               .with_owner('root')
               .with_group('foreman')
               .with_mode('0644')
-              .with_content(/:concurrency: 1/)
-              .with_content(/:queues:\n  - default\n  - remote_execution/)
+              .with_content(expected_content)
           }
           it {
             should contain_service('dynflow-sidekiq@test_worker')
               .with_ensure('running')
               .with_enable(true)
           }
+
+          context 'and priorities' do
+            let(:params) do
+              {queues: [['default', 1], ['remote_execution', 1]]}
+            end
+
+            let(:expected_content) do
+              <<~CONTENT
+                ---
+                :concurrency: 1
+                :queues:
```

Comment:
`queues` can either be a list or a list of lists? Thats right ugly but I realize not our "concern" per this feature.

Reply:
Yes, that is indeed my impression based on @adamruzicka's patch. I think we're just exposing what Dynflow accepts.

@adamruzicka can you confirm?

Reply:
It is a sidekiq-specific thing[1], Dynflow has no role in this. It can be either a list or a list of two element lists, where first element is name of the queue and the other is the priority.

[1] - https://github.com/mperham/sidekiq/wiki/Advanced-Options#queues